types/xovis/PC2SV5/view_direction.schema.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$id": "https://akenza.io/xovis/pc2s/view_direction.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "View direction",
+  "type": "object",
+  "properties": {
+    "xCoordinate": {
+      "title": "X Coordinate",
+      "description": "X Coordinate",
```

Comment:
X coordinate of the view direction? how is the coordinate system defined?

Reply:
Depending on the sensor field of view. Is different from sensor to sensor.

Reply:
ok but it is important to know whether the 0 is at the top or bottom left, please find that out and add the information